components/admin_console/manage_teams_modal/manage_teams_modal.jsx

```diff
@@ -101,6 +101,10 @@ export default class ManageTeamsModal extends React.Component {
         }
     }
 
+    handleMemberChange = async () => {
```

Comment:
Do we need async here?

Reply:
I don't actually know! In that method I'm calling directly another `async` method, so I assumed this one had to be `async` too, but perhaps that is an incorrect assumption?

Reply:
@grundleborg I checked and you don't need async here. so, we can safely remove it
Here you don't need to await for any async actions so we can just directly call `getTeamMembers `.

Reply:
Done.

Reply:
just one more thing while we are at it. Incase if you needed async you would have to have a `await` inside the function or return the async function inside to await any function that calls this async function.

Reply:
Thanks dude. Really appreciate you sharing your javascript knowledge, as I'm not very familiar with these bits of the language :)